reducers/windows.js

```diff
@@ -106,6 +110,19 @@ export default function windows(state = defaultState, action) {
             mapMargins: mapMargins
         };
     }
+    case SET_MENU_MARGIN: {
+        const menuMargins =  {
+            right: action.right,
+            left: action.left
+        };
+        const mapMargins =  {
+            right: action.right,
+            bottom: 0,
+            left: action.left,
+            top: 0
+        };
```

Comment:
This looks wrong - why reset the top and bottom map margin to zero?

Reply:
Yes, I correct that